slices/postgresql-client-16.yaml

```diff
@@ -0,0 +1,31 @@
+package: postgresql-client-16
+
+essential:
+  - postgresql-client-16_copyright
+
+slices:
+  bins:
+    contents:
+      /usr/bin/createdb: {symlink: /usr/lib/postgresql/16/bin/createdb}
+      /usr/bin/dropdb: {symlink: /usr/lib/postgresql/16/bin/dropdb}
```

Comment:
These two commands seem like they might be needed less than `psql` or `pg_dump`, so it might be worth considering if we want to put those into separate slices.